Open Weather Map (OWM) Segment Not Honoring Custom HTTP_Timeout and Cache_Timeout Properties

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I've increased the `http_timeout` and `cache_timeout` defaults for the OWM segment, but they do not appear to be applying.

Here is the relevant segment configuration:

```yaml
blocks:
  - alignment: right
    segments:
      - background: '#575656'
        foreground: '#D6DEEB'
        properties:
          apikey: '[api_key]'
          location: '[location_query]'
          units: 'imperial'
          http_timeout: 200
          cache_timeout: 15
        style: diamond
        leading_diamond: 
        trailing_diamond: 
        template: "{{ .Weather }}  {{ .Temperature }}{{ .UnitIcon }}\u2800"
        type: owm
    type: prompt
```

_Note: Location and ApiKey properties intentionally omitted for privacy and security._

### Theme

I'm using a customized theme with a block configuration.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
[DEBUG] 10:48:17.141 block.go:Debug:230 → Segment: owm
[TRACE] 10:48:17.141 shell.go:Pwd() - 0s
[DEBUG] 10:48:17.141 shell_windows.go:56 → C:\Users\bmkaiser
[TRACE] 10:48:17.141 shell.go:GOOS() - 0s
[DEBUG] 10:48:17.141 properties.go:GetString:28 → imperial
[DEBUG] 10:48:17.141 properties.go:GetInt:40 → cache_timeout: 10
[DEBUG] 10:48:17.141 properties.go:GetString:28 → [api_key]
[DEBUG] 10:48:17.141 properties.go:GetString:28 → [location_query]
[DEBUG] 10:48:17.141 properties.go:GetString:28 → imperial
[DEBUG] 10:48:17.141 properties.go:GetInt:40 → http_timeout: 20
[DEBUG] 10:48:17.141 shell.go:HTTPRequest:686 ↓
    GET /data/2.5/weather?q=[location_query]&units=imperial&appid=[api_key] HTTP/1.1
    Host: api.openweathermap.org
    User-Agent: Go-http-client/1.1
    Accept-Encoding: gzip


[ERROR] 10:48:17.162 shell.go:HTTPRequest:690 → Get "http://api.openweathermap.org/data/2.5/weather?q=[location_query]&units=imperial&appid=[api_key]": context deadline exceeded
[TRACE] 10:48:17.162 shell.go:HTTPRequest(http://api.openweathermap.org/data/2.5/weather?q=[location_query]&units=imperial&appid=[api_key]) - 21.3491ms
```

--- a/.vscode/launch.json
+++ b/.vscode/launch.json
@@ -12,8 +12,7 @@
         "print",
         "primary",
         "--shell=pwsh",
-        "--terminal-width=200",
-        "--config=/Users/jan/.test.omp.json"
+        "--terminal-width=200"
       ]
     },
     {

--- a/src/properties/map.go
+++ b/src/properties/map.go
@@ -123,6 +123,8 @@ func (m Map) GetInt(property Property, defaultValue int) int {
 		return v
 	case int64:
 		return int(v)
+	case uint64:
+		return int(v)
 	case float64:
 		intValue, ok := val.(float64)
 		if !ok {

